tutorials/mongoose/updateMany.md

```diff
@@ -0,0 +1,37 @@
+If you wish to update several documents in your model with mongoose, you should use the `updateMany()` function.
+It takes up to four parameters:
+
+- **filter**, what documents it should find that match the filter.
+You can omit this parameter if you want to update all the documents in the model.
+- **update**, what the documents that are being updated should now contain.
+- **options**, modifications that can be made to the process of the function call.
+- **A callback function(err, res)**, where `res` has five properties: `acknowledged`, `modifiedCount`, `upsertedId`, `upsertedCount`, and `matchedCount`.
+Add this argument if you are not using `async/await`.
+
+```javascript
+const testSchema = new mongoose.Schema({
+    name: String,
+    location: String
+});
+
+const Test = mongoose.model('Test', testSchema);
+
+await Test.create({name: 'Test Testerson'});
+await Test.create({name: 'Masteringjs'});
+await Test.create({name: 'MeanIT'});
+
+await mongoose.updateMany({},{location: 'Florida'},{}) // {acknowledged, modifiedCount, upsertedId, upsertedCount, matchedCount}
+```
+
+## Return Value
+
+Mongoose will return five properties from the `updateMany()` call: `acknowledged`, `modifiedCount`, `upsertedId`, `upsertedCount`, and `matchedCount`.
```

Comment:
Make this a [ordered markdown list](https://www.markdownguide.org/basic-syntax/#lists-1) of properties